scripts/popup.js

```diff
@@ -11,9 +13,19 @@ document.addEventListener("DOMContentLoaded", () => {
           document.getElementById(element.id).checked = element.checked;
         }
       });
+
+      const hideDeleteToggle = elements.find(el => el.id === "hide-delete-toggle");
```

Comment:
there's a nasty forEach right above, could you use it to avoid looping 2 times on elements ? and replace it by a `for (element of elements)` please btw, forEach is a mess in terms of performance

Reply:
I reckon that should be included in a separate commit that is about performance, because otherwise, if this doesn't get merged we won't get the optimization.